rali/rali/source/rali_api_augmentation.cpp

```diff
@@ -448,7 +468,9 @@ raliResize(
         // For the nodes that user provides the output size the dimension of all the images after this node will be fixed and equal to that size
         output->reset_image_roi();
 
-        context->master_graph->add_node<ResizeNode>({input}, {output});
+        std::shared_ptr<ResizeNode> resize_node =  context->master_graph->add_node<ResizeNode>({input}, {output});
```

Comment:
Codacy found an issue: [Either the condition '!context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5976200)